### PR TITLE
mobile view

### PR DESCRIPTION
**Pull request by contactsanthosh21**
- Added mobile view
 - Fixed ipad alignments

### CODE REVIEW

File: src/Pages/AboutUs.jsx [hunk 1]
```diff
@@ -12,7 +12,7 @@ const AboutUs = () => {
     return (
         <>
             <Values />
-            <Quote quoteContent={aboutQuoteContent} quoteAuthor={aboutQuoteAuthor} />
```
Comment (ankitg32) [inline src/Pages/AboutUs.jsx:15]: why is this needed?

File: src/sections/ContactUs/ContactUs.jsx [hunk 2]
```diff
@@ -24,7 +24,7 @@ function useEmail(formId){
     e.preventDefault();
     const form = document.getElementById(formId)
     const formData = new FormData(form)
-    const url = 'https://formsubmit.co/72bebc68699d2efa6a8744eabeb45b2d';
```
Comment (ankitg32) [inline src/sections/ContactUs/ContactUs.jsx:27]: TODO: ask the person who this email belongs to, to give us the encoded string to be used in the code. this is important to prevent his email from being publicly exposed

File: src/sections/Header/header.scss [hunk 3]
```diff
@@ -82,4 +82,57 @@
...
+    .header__nav__mobile{
+        display: block;
+    }
+    .bm-burger-bars{
+        background: $color-black;
+        width: 10%;
+        height: 3%!important;
+        right: 8% !important;
+        left: auto!important;
+        &:nth-child(1){
```
Comment (ankitg32) [inline src/sections/Header/header.scss:106]: Little unsure of why we have to hard code children and also override top with !important. Is there a better, and less rigid way to do this?

File: src/sections/Header/header.scss
Comment (ankitg32) [inline src/sections/Header/header.scss:1]: too many `!important`s are making me unsure for this

File: src/styles.scss [hunk 4]
```diff
@@ -17,7 +17,7 @@ p {
 
 .container {
     position: relative;
-    width: 100vw;
+    width: fit-content;
```
Comment (ankitg32) [inline src/styles.scss]: for very big screens, the container will become smaller and not fit the complete screen. I'd say reconsider applying this css this way. One suggestion is to apply this to smaller screen sizes using media query.

File: src/sections/Planning/PlanningCardHover.scss
Comment (ankitg32) [inline src/sections/Planning/PlanningCardHover.scss:1]: too many `!important`s...

**Review by ankitg32 — COMMENTED** [review]
Primarily, it feels like css is too rigid because of heavy use of !important. It might be possible to avoid this.
other small suggestions

ALSO, CAN WE CLOSE THE HAMBURGER MENU ONCE THE USER HAS MADE A SELECTION?

File: src/Pages/AboutUs.jsx (hunk 1 above)
Comment (contactsanthosh21) [reply src/Pages/AboutUs.jsx:15]: the quote banner not fits in to entire screen or same css not working for both..so passing this custom class to differentiate

File: src/sections/ContactUs/ContactUs.jsx (hunk 2 above)
Comment (contactsanthosh21) [reply src/sections/ContactUs/ContactUs.jsx:27]: Will do before merging

File: src/sections/Planning/PlanningCardHover.scss
Comment (contactsanthosh21) [reply src/sections/Planning/PlanningCardHover.scss:1]: For now will go with this style override approach

File: src/sections/Header/header.scss
Comment (contactsanthosh21) [reply src/sections/Header/header.scss:1]: For now will go with this style override approach

File: src/sections/Header/header.scss (hunk 3 above)
Comment (contactsanthosh21) [reply src/sections/Header/header.scss:106]: For now will go with this style override approach

**Review by ankitg32 — APPROVED** [review]
